Sources/Starknet/Providers/StarknetProviderProtocol.swift

```diff
@@ -26,4 +45,14 @@ public extension StarknetProviderProtocol {
     func callContract(_ call: StarknetCall) async throws -> [Felt] {
         return try await callContract(call, at: defaultBlockId)
     }
+    
+    /// Get nonce of given starknet contract at latest block.
+    ///
+    /// - Parameters
+    ///     - contract: address of a contract, for which the nonce should be returned.
+    ///
+    /// - Returns: Felt value of contract's current nonce.
+    func getNonce(of contract: Felt) async throws -> Felt {
+        return try await getNonce(of: contract, at: defaultBlockId)
+    }
```

Comment:
How is the support for default parameters in swift. Perhaps we could do `at blockId = somehow default to latest/pending block` instead of polymorphic methods?

I wonder if that would be better UX wise?

Reply:
The problem is that default parameters are not allowed in protocols (intefaces)